### PR TITLE
Fix miscellaneous things

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: remote-trait-object/src/service/serde_support.rs [hunk 1]
```diff
@@ -80,8 +80,9 @@ impl<T: ?Sized + Service> SRwLock<T> {
     }
```
Comment (majecty) [inline remote-trait-object/src/service/serde_support.rs:80]: In the commit mesesage: Edit old comments on **prot**_thread_local -> Edit old comments on **port**_thread_local

File: remote-trait-object/src/service.rs [hunk 2]
```diff
@@ -32,13 +32,6 @@ pub type MethodId = u32;
 #[derive(PartialEq, Serialize, Deserialize, Debug, Clone, Copy)]
 pub struct HandleToExchange(pub(crate) ServiceObjectId);
```
Comment (majecty) [inline remote-trait-object/src/service.rs:34]: How about changing the commit title from "Remove new_singleton()" to "Remove unused new_singleton()"?

Comment (junha1) [reply remote-trait-object/src/service.rs:34]: done

File: remote-trait-object/src/service/serde_support.rs (hunk 1 above)
Comment (junha1) [reply remote-trait-object/src/service/serde_support.rs:80]: done

**Pull request merged**